FinSearchUnified/Bundle/SearchBundleFindologic/FacetHandler/ImageFacetHandler.php

```diff
@@ -125,25 +129,27 @@ private function getMediaListItems(array $actives, SimpleXMLElement $filterItems
         }
 
         $options = [
-            'complete' => function (CompleteEvent $event) use ($items, &$listItems) {
+            'complete' => function (CompleteEvent $event) use ($items, $orderedArray, &$listItems) {
```

Comment:
Don't add the type of a variable, into it's name. It should be clear by the name of the variable, which contents are inside. For example `$orderedArray` => `$orderedUsers`.